persistence/src/main/resources/demoSubmissions.csv

```diff
@@ -1,4 +1,6 @@
-0|2017|1|created|1484842271000|0
-0|2017|2|created|1484842271000|0
-0|2017|3|created|1484842271000|0
-1|2017|1|created|1484842271000|0
+2|2017|1|parsed-with-errors|1484842271000|0
```

Comment:
Can we have a submission here with a status of created?  When testing any endpoints that require a file to be uploaded, it would be easier to have a submission ready to go, rather than having to create a new one every time we re-run the API.